glide.lock

```diff
@@ -2133,7 +2125,7 @@ imports:
   subpackages:
   - config/v1beta1
 - name: k8s.io/kubernetes
-  version: a3d846c6abdf03e54525c3ebca35c40441462261
+  version: 3538c7842b1e43174e181b732cd3147a56711427
```

Comment:
how did you do this bump without overwriting the changes of the UPSTREAM commit? Did you just leave out the kube files in the bump, i.e. these SHAs are now wrong?